tests/publiccloud/create_aistack_env.pm

```diff
@@ -93,45 +98,57 @@ sub config_kubectl {
 }
 
 sub install_aistack_chart {
-    my ($instance, $ai_chart_repo, $namespace, $vf_name) = @_;
+    my ($instance, $namespace) = @_;
     my $SECRET_application_collection = get_var('_SECRET_DOCKER');
     my $cert_repo = get_var('HELM_CERTS');
     my $ingress_repo = get_var('HELM_INGRESS');
     my $docker_user_name = get_var('USER_DOCKER');
     my $repo_url = get_var('HELM_CHARTS');
-    my $git_token = get_var('_SECRET_SSH');
     my $ing_ver = get_var('ING_VERSION');
     my $local_storage_name = 'local_path_storage.yaml';
+    my $milvus_custom = 'milvus_custom.yaml';
+    my $milvus_values = 'milvus_values.yaml';
+    my $ollama_values = 'ollama_values.yaml';
+    my $openwebui_values = 'open_webui_values.yaml';
+    my $milvus_helm_repo = 'oci://dp.apps.rancher.io/charts/milvus';
+    my $ollama_helm_repo = 'oci://dp.apps.rancher.io/charts/ollama';
+    my $openwebui_helm_repo = 'oci://dp.apps.rancher.io/charts/open-webui';
```

Comment:
Please consider if all those paraemters would be better as **external parameters** settings in a yaml, then loaded here as multiple lines like:
```
my $var_xxxx = get_required_var('AI_xxxx'); 
...
```

Reply:
MR 1989 was merged ok :slightly_smiling_face: